code/game/machinery/dance_machine.dm

```diff
@@ -119,7 +123,7 @@
 		data["track_selected"] = selection.song_name
 		data["track_length"] = DisplayTimeText(selection.song_length)
 		data["track_beat"] = selection.song_beat
-	data["volume"] = volume
+	data["volume"] = min(volume, 100) // funny hrefsploit
```

Comment:
this is ui data what exploit are you gonna get here

Reply:
lmao sorry wrong spot

Reply:
lol